flytekit/types/structured/basic_dfs.py

```diff
@@ -143,6 +147,8 @@ def encode(
         structured_dataset: StructuredDataset,
         structured_dataset_type: StructuredDatasetType,
     ) -> literals.StructuredDataset:
+        import pyarrow.parquet as pq
```

Comment:
Is it possible to lazily load this module instead of importing it these methods?

Reply:
we can't. we can only pass module name to `lazy_module`. like `lazy_module(pyarrow)`